search-app/src/cmr/search/site/routes.clj

```diff
@@ -0,0 +1,28 @@
+(ns cmr.search.site.routes
+  "This namespace is the one responsible for the routes intended for human
```

Comment:
nitpick: I think I would rewrite this as "routes for static resources" (I'm assuming these are not generated upon request), since it includes CSS, etc., that aren't generally consumed by humans.